core/infra/config/peer_service_with_json.cpp

```diff
@@ -113,13 +114,26 @@ std::string PeerServiceConfig::getMyIp() {
 }
 
 double PeerServiceConfig::getMaxTrustScore() {
-  return this->getMaxTrustScoreWithDefault(1.0); // WIP to support trustRate = 1.0
+  return this->getMaxTrustScoreWithDefault(10.0); // WIP to support trustRate = 10.0
 }
 
 size_t PeerServiceConfig::getMaxFaulty() {
   return std::max( 0, ((int)this->getPeerList().size()-1) / 3 );
 }
 
+bool PeerServiceConfig::isMyActive() {
```

Comment:
`PeerServiceConfig::isMyActive` is going to be `peer::myself::isActive`.
This changing is applied by `refactor/develop` branch. 
So, this branch will not repair its name.